### ISSUE TITLE
Question on Normalization during training and test

### ISSUE THREAD

**Issue by ramsever** [hysts/pytorch_cutout#1]
It seems that the normalization during training is different then the normalization during test.
why define:
https://github.com/hysts/pytorch_cutout/blob/master/dataloader.py#L24
When you can use:
https://github.com/hysts/pytorch_cutout/blob/master/dataloader.py#L86
